assets/src/story-embed-block/block/edit.js

```diff
@@ -170,7 +170,7 @@ function StoryEmbedEdit({
   const onResizeStart = () => toggleSelection(false);
   const onResizeStop = () => toggleSelection(true);
 
-  const label = __('Web Story URL', 'web-stories');
+  const label = __('Story URL', 'web-stories');
```

Comment:
Still don't understand this change?

Reply:
Sorry, you have to review this part again, some of these changes were part of previously created one monolith PR.
https://github.com/rtCamp/web-stories-wp/pull/150#discussion_r543569065

Reply:
In design reviews it was flagged that we should name this just "Story URL" and I said it's fine to make the change here as well because the same component is being re-used.